Gemfile

```diff
@@ -11,20 +11,21 @@ end
 
 group :development do
   gem 'sqlite3'
+  gem 'web-console', '~> 2.0'
+  gem 'rails_db'
 end
 
 gem 'sass-rails', '~> 5.0'
 gem 'uglifier', '>= 1.3.0'
 gem 'coffee-rails', '~> 4.1.0'
 gem 'jquery-rails'
 gem 'turbolinks'
-gem 'bootstrap'
+gem 'bootstrap-sass'
 gem 'figaro'
 gem 'pry'
```

Comment:
@ianroberts131 Use pry-rails instead of pry in the `Gemfile` and ensure its isolated to the development+test groups. 

At the moment pry will be loaded in production (and dev and test) which is an unnecessary overhead for the app. 

As a rule of thumb, only run debugging tools like pry temporarily in a production environment when its absolutely necessary to diagnose a production issue.